### PR TITLE
Add admin climate recommendations

### PR DESCRIPTION
**Pull request by arnaud-morvan**
@pgiraud : can you review.

### CODE REVIEW

File: thinkhazard/__init__.py [hunk 1]
```diff
@@ -44,11 +44,21 @@ def main(global_config, **settings):
...
     config.add_route('admin_technical_rec', '/admin/technical_rec')
     config.add_route('admin_technical_rec_new', '/admin/technical_rec/new')
     config.add_route('admin_technical_rec_edit', '/admin/technical_rec/{id}')
+
     config.add_route('admin_admindiv_hazardsets',
                      '/admin/admindiv_hazardsets/{hazardtype:([A-Z]{2})}')
+
+    config.add_route('admin_climate_rec', '/admin/climate_rec')
+    config.add_route('admin_climate_rec_hazardtype',
+                     '/admin/climate_rec_{hazard_type}')
```
Comment (pgiraud) [inline thinkhazard/__init__.py]: Any reason why you didn't write it the same way as we already do for other routes?
ie. '/admin/climate_rec/{hazardtype:([A-Z]{2})}'

Comment (arnaud-morvan) [inline thinkhazard/__init__.py]: Yes it seems better, I will change for it.

File: thinkhazard/tests/views/test_admin.py [hunk 2]
```diff
@@ -36,7 +44,7 @@ def test_hazardcategory(self):
     def test_technical_rec(self):
         resp = self.testapp.get('/admin/technical_rec', status=200)
         records = resp.html.select('.item-technicalrecommendation')
-        self.assertEqual(len(records), 6)
+        self.assertEqual(len(records), 2)
```
Comment (pgiraud) [inline thinkhazard/tests/views/test_admin.py]: Why did this change?

File: thinkhazard/templates/admin/climate_rec_form.jinja2 [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            name="associations"
+            {{ 'checked' if current_rec.id == obj.id else '' }}
+            value="{{ admin_div.id }}"> {{ admin_div.name }}</input>
+          {% if (current_rec and current_rec.id != obj.id) %}
+          &nbsp;
+          <a
+            href="{{ request.route_url('admin_climate_rec_edit', id=current_rec.id) }}"
+            title="{{ current_rec.text }}"
+            ><span class="current_rec text-muted"><i>{{ current_rec.text }}</i></span></a>
+          {% endif %}
```
Comment (pgiraud) [inline thinkhazard/templates/admin/climate_rec_form.jinja2]: It would be nice to be able to click on text.

Comment (arnaud-morvan) [inline thinkhazard/templates/admin/climate_rec_form.jinja2]: Labels are now clickable. Waiting for travis ...

**Comment by pgiraud** [comment]
Merging on behalf of @arnaud-morvan.

**Pull request merged**